test/Microsoft.ML.Tests/TensorFlowEstimatorTests.cs

```diff
@@ -135,8 +135,7 @@ void TestOldSavingAndLoading()
         [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 output differs from Baseline
         void TestCommandLine()
         {
-            var env = new MLContext();
-            IHostEnvironment environment = env;
+            IHostEnvironment environment = new MLContext();
```

Comment:
I'm not sure what this code is doing... we are creating an environment, but not doing anything with it.

Reply:
i am using the environment to register the assmebly that is being used by maml. The maml was not able to load this dependency by itself on netfx.

Reply:
It doesn't make sense to `using the environment to register the assmebly` because you are not using that Environment in the Maml command.

If you just want to ensure the assembly is loaded, you can just reference the `typeof(blah)`. But I think we should be using that `environment` in the `Maml` command.

Reply:
I tried exposing and using Maml.MainCore function but then it resulted in adding an assembly Registrar https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Maml/MAML.cs#L61
(which is the only major diff between main and maincore)
so I ended up just adding the typeof (I was earlier worried about these statements being removed in the release mode. but its working)